### ISSUE TITLE
Talkback issues with quotes

### ISSUE THREAD

**Issue by nikclayton** [pachli/pachli-android#2049]
@dhamlinmusic@dragonscave.space reports:

> So quoted posts are appearing as what talkback reads as "detected image post content" below the post by the person quoting them, but have no indication they are a quote, they just show up like a post you cannot interact with that talkback sees as an embedded image.
